platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminContext.java

```diff
@@ -289,8 +293,13 @@ private ImportMigrationConfigurationAdminEntry proxy(ImportMigrationEntry entry)
             .record(
                 new MigrationException("Import error: missing pid from configuration [%s].", path));
       } else {
-        final String bundleLocation =
-            Objects.toString(properties.remove(ConfigurationAdmin.SERVICE_BUNDLELOCATION), null);
+        String bundleLocation = null;
```

Comment:
✏️ As opposed to put the conversion logic here, I would recommend that the constructor of this class receives a `Function<Dictionary<String, Object>, Dictionary<String, Object>> convertor` which you store and just above here where we do `properties = readProperties(entry, ps, is)`, do instead `properties = convertor.apply(process.readProperties(entry, ps, is));`. 

This allows the logic for conversion to remain inside the migratable where you either pass `Function.identity()` or a function that does the testing and property removal you are doing here. The day you need to do something different based on different version or different bundle, you will be able to keep all that logic in one place.